internal/persistence/persistors/mysqlstore/store_organization_test.go

```diff
@@ -0,0 +1,497 @@
+		QueryTimeouts: testQueryTimeouts,
+	}
+
+	m, err := New(cfg)
+	require.NoError(t, err, "unexpected error")
+	require.NotNil(t, m, "unexpected nil")
+
+	txHandler, err := mysqltx.New(&mysqltx.Config{
+		Logger:       testLogger,
+		Db:           db,
+		DatabaseName: cp.Database,
+	})
+	require.NoError(t, err, "unexpected error creating the tx handler")
+
+	txHandlerDb, err := txHandler.Db(testCtx)
+	require.NoError(t, err, "unexpected error fetching the db from the tx handler")
+	require.NotNil(t, txHandlerDb, "unexpected nil tx handler db")
+
+	mutations := func(t *testing.T, db *sqlx.DB) int {
+		entry := mysqlmodel.Organization{
+			ID:            1,
+			Name:          "TEST",
+			CreatedBy:     null.IntFrom(2),
+			LastUpdatedBy: null.IntFrom(3),
+			CreatedAt:     time.Now(),
+			LastUpdatedAt: null.TimeFrom(time.Now()),
+			IsActive:      true,
+		}
+		err := entry.Insert(context.Background(), db, boil.Infer())
+		require.NoError(t, err, "error inserting sample data")
+		return entry.ID
+	}
+
+	orgID := mutations(t, db)
+
+	paginatedOrganizations, err := m.GetOrganizations(testCtx, txHandlerDb, nil)
+	require.NoError(t, err, "unexpected error fetching the organizations from the database")
+	require.NotNil(t, paginatedOrganizations, "unexpected nil organizations")
+	require.True(t, len(paginatedOrganizations.Organizations) > 0, "unexpected empty organizations")
+
+	updateOrganization := model.UpdateOrganization{
+		Id: orgID,
+		Name: null.String{
+			String: paginatedOrganizations.Organizations[0].Name + " new",
+			Valid:  true,
+		},
+	}
+
+	cat, err := m.UpdateOrganization(testCtx, txHandlerDb, &updateOrganization)
+	require.NoError(t, err, "unexpected error updating the organization")
+	assert.Equal(t, paginatedOrganizations.Organizations[0].Name+" new", cat.Name)
+}
+
+func Test_UpdateOrganization_Fail(t *testing.T) {
+	db, cp, cleanup := mysqlhelper.TestGetMockMariaDB(t)
+	defer cleanup()
+
+	cfg := &Config{
+		Logger:        testLogger,
+		QueryTimeouts: testQueryTimeouts,
+	}
+
+	m, err := New(cfg)
+	require.NoError(t, err, "unexpected error")
+	require.NotNil(t, m, "unexpected nil")
+
+	txHandler, err := mysqltx.New(&mysqltx.Config{
+		Logger:       testLogger,
+		Db:           db,
+		DatabaseName: cp.Database,
+	})
+	require.NoError(t, err, "unexpected error creating the tx handler")
+
+	txHandlerDb, err := txHandler.Db(testCtx)
+	require.NoError(t, err, "unexpected error fetching the db from the tx handler")
+	require.NotNil(t, txHandlerDb, "unexpected nil tx handler db")
+
+	updateOrganization := model.UpdateOrganization{
+		Id: 3123123123,
+		Name: null.String{
+			String: "Non-existent Organization",
+			Valid:  true,
+		},
+	}
+
+	_, err = m.UpdateOrganization(testCtx, txHandlerDb, &updateOrganization)
+	require.Error(t, err, "expected error when updating a non-existent organization")
+	assert.Contains(t, err.Error(), "expected exactly one entry for entity: 3123123123")
+}
+
+type createOrganizationTestCase struct {
```

Comment:
and rename all testCase structs to passive voice... 

e.g from: createOrganizationTestCase to **testCaseCreateOrganization**

Reply:
The structs have now been renamed correctly.